tests_e2e/test_e2e.py

```diff
@@ -8,11 +9,15 @@
 from todo_app.flask_config import Config
 from todo_app.app import create_app
 
+
+
 @pytest.fixture(scope="module")
 def driver():
-    with webdriver.Firefox() as driver:
+    with webdriver.Chrome("./chromedriver") as driver:
```

Comment:
Minor but this won't work on Windows - you can avoid that here by just removing the argument completely (`with webdriver.Chrome() as driver:`) and asking the user to have their Chromedriver executable on their PATH, but it is easier to keep this here for running the tests in Docker so it might be something we just want to accept for now!